clvm_tools/NodePath.py

```diff
@@ -71,7 +71,9 @@ def __init__(self, index=1):
         self._index = index
 
     def as_short_path(self):
-        return self._index
+        index = self._index
+        byte_count = (index.bit_length()+7) >> 3
```

Comment:
I think `/ 8` is a clearer expression of intent than `>> 3`. but I suppose it's a matter of taste